plugins/hls-import-actions-plugin/src/Ide/Plugin/ImportActions.hs

```diff
@@ -110,28 +110,31 @@ runImportCommand recorder ideState eird@(ResolveOne _ _) = pluginResponse $ do
           logWith recorder Error (LogWAEResponseError re)
           pure ()
         logErrors (Right _) = pure ()
-runImportCommand _ _ (ResolveAll _) = do
-  pure $ Left $ ResponseError (InR ErrorCodes_InvalidParams) "Unexpected argument for command handler: ResolveAll" Nothing
+runImportCommand _ _ rd = do
```

Comment:
It's weird that we don't have `RefineOne`, especially since we have the per-import lens, but I guess we didn't before either.